pkgs/os-specific/linux/zfs/bcachefs-compat.patch

```diff
@@ -0,0 +1,16 @@
+diff --git c/include/os/linux/spl/sys/kmem_cache.h i/include/os/linux/spl/sys/kmem_cache.h
+index cc9cafa84..862a2dbc1 100644
+--- c/include/os/linux/spl/sys/kmem_cache.h
++++ i/include/os/linux/spl/sys/kmem_cache.h
+@@ -198,7 +198,11 @@ extern uint64_t spl_kmem_cache_entry_size(kmem_cache_t *cache);
+     spl_kmem_cache_create(name, size, align, ctor, dtor, rclm, priv, vmp, fl)
+ #define	kmem_cache_set_move(skc, move)	spl_kmem_cache_set_move(skc, move)
+ #define	kmem_cache_destroy(skc)		spl_kmem_cache_destroy(skc)
++
++/* This is necessary to be compatible with bcachefs for example. */
++#undef  kmem_cache_alloc
```

Comment:
upstream?

Reply:
I will upstream it yes

Reply:
https://github.com/openzfs/zfs/pull/15143

Reply:
This is upstreamed in 2.1.13 and 2.2.x.